### PR TITLE
Added rubocop to cqm-models; Fixed rubocop issues

### PR DESCRIPTION
**Pull request by holmesie**
Added rubocop to cqm-models; Fixed rubocop issues

Pull requests into cqm-models require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/TACO-32
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name: @okeefm
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: app/assets/javascripts/index.js [hunk 1]
```diff
@@ -1,122 +1,122 @@
...
-module.exports.ProcedureOrderSchema = require('./ProcedureOrder.js').ProcedureOrderSchema;
-module.exports.MedicationOrder = require('./MedicationOrder.js').MedicationOrder;
-module.exports.MedicationOrderSchema = require('./MedicationOrder.js').MedicationOrderSchema;
-module.exports.SubstanceRecommended = require('./SubstanceRecommended.js').SubstanceRecommended;
-module.exports.SubstanceRecommendedSchema = require('./SubstanceRecommended.js').SubstanceRecommendedSchema;
-module.exports.InterventionRecommended = require('./InterventionRecommended.js').InterventionRecommended;
-module.exports.InterventionRecommendedSchema = require('./InterventionRecommended.js').InterventionRecommendedSchema;
-module.exports.PhysicalExamPerformed = require('./PhysicalExamPerformed.js').PhysicalExamPerformed;
-module.exports.PhysicalExamPerformedSchema = require('./PhysicalExamPerformed.js').PhysicalExamPerformedSchema;
+module.exports.["Patient", [{:name=>"birthDatetime", :type=>"System.DateTime"}]] = require('./["Patient", [{:name=>"birthDatetime", :type=>"System.DateTime"}]].js').["Patient", [{:name=>"birthDatetime", :type=>"System.DateTime"}]];
```
[the remark this answers is not in the document]
Comment (holmesie) [reply app/assets/javascripts/index.js]: I am not sure why this is changing, I did not touch anything JS wise. @ljades can you explain why this file is changing?

File: app/models/qdm/adverse_event.rb [hunk 2]
```diff
@@ -1,12 +1,15 @@
-class QDM::AdverseEvent < QDM::DataElement
```
[the remark this answers is not in the document]
Comment (holmesie) [reply app/models/qdm/adverse_event.rb]: This was the reason the tests were failing. Fixed.

File: app/assets/javascripts/index.js [hunk 3]
```diff
@@ -1,122 +1,122 @@
 module.exports.Result = require('./Result.js').Result;
 module.exports.ResultSchema = require('./Result.js').ResultSchema;
-module.exports.Patient = require('./Patient.js').Patient;
-module.exports.PatientSchema = require('./Patient.js').PatientSchema;
```
[the remark this answers is not in the document]
Comment (holmesie) [reply app/assets/javascripts/index.js]: See previous comment, I'm not sure what is happening here. I think the generator script might have been updated without commiting the files. @ljades any thoughts here?
